src/seqan2seqan3.hpp

```diff
@@ -268,6 +271,14 @@ inline auto score(Score<TValue, ScoreMatrix<TSequenceValue, TSpec>> const & sche
     return score(scheme, AminoAcid{seqan3::to_char(a1)}, AminoAcid{seqan3::to_char(a2)});
 }
 
+template <typename TValue, seqan3::nucleotide_alphabet alph_t>
+inline auto score(Score<TValue, ScoreMatrix<Dna5, BisulfiteMatrix>> const & scheme,
+                  alph_t const a1,
+                  alph_t const a2)
+{
+    return score(scheme, Dna5{seqan3::to_char(a1)}, Dna5{seqan3::to_char(a2)});
```

Comment:
Are both intentionally converted to Dna5?

Reply:
I made the bisulfite scheme for Dna5 sequences .. I thought this needs to be done then?

Reply:
Ah, true. Scoring happens on unreduced and that is always Dna5.